web/.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn --cwd web run lint-staged
+cd web yarn run type && yarn run lint-staged 
```

Comment:
Is there any special reason for changing `yarn --cwd` to `cd web yarn`? I think it can work in both way, but yarn --cwd is more self-contained, it doesn't alter shell's working directory and could avoid side effects of that. (although we only have one cmd in this case but i think it's still ideal)

Reply:
Thanks. You're right, `yarn --cwd` doesn't change the working directory. I've updated to use that instead. Please note that the command has been repeated as it might not be possible to chain the commands into one ( need to specify `yarn --cwd` in the second command as well ).

Reply:
Got it, thank you!